components/director/internal/authenticator/consumer_ctx.go

```diff
@@ -0,0 +1,28 @@
+package authenticator
+
+import (
+	"context"
+	"errors"
+
+	"github.com/kyma-incubator/compass/components/director/internal/tenantmapping"
+)
+
+const ConsumerKey = "consumer type"
+
+var NoConsumerError = errors.New("cannot read consumer from context")
```

Comment:
Think about moving this file to `internal/consumer` package, similarly to `tenant`